base/iostream.jl

```diff
@@ -321,13 +321,18 @@ function peek(s::IOStream)
     ccall(:ios_peekc, Cint, (Ptr{Void},), s)
 end
 
+
+function _seekback(pos::Integer, s::IOStream)
```

Comment:
So it is, I was mimicking my C `ios_seekback` function which in turn was mimicking `ios_ungetc` which I guess was mimicking the standard `ungetc`. I blame Dennis Ritchie.

Should I change my C function as well?